Views/Storyboards/Base.lproj/Main.storyboard

```diff
@@ -191,11 +191,11 @@
                         <outlet property="navigationBar" destination="OwN-92-HaX" id="mZx-So-UkL"/>
                         <outlet property="synchroLabel" destination="qI5-Ep-5Qd" id="DTW-xd-421"/>
                         <outlet property="tableView" destination="ttg-Mf-cp2" id="vuV-KO-27u"/>
-                        <segue destination="pQK-Bk-vqJ" kind="showDetail" identifier="addIntervention" id="74J-jI-YTr"/>
-                        <segue destination="krK-gd-8UN" kind="presentation" identifier="SegueLogOut" id="a8t-Pd-Dxa"/>
+                        <segue destination="pQK-Bk-vqJ" kind="showDetail" identifier="ShowAddInterventionVC" id="IIb-81-RCP"/>
```

Comment:
Segue identifiers should be written in camelCase, with a first letter lowercased